monero-outreach-docs/translations/es/archive/rpc_pay_es.md

```diff
@@ -1,113 +1,109 @@
+RPC-Pay es una nueva característica añadida al _monerod_ y los monederos que habilitan pagos para las llamadas RPC, utilizando hashes de minería. Esta capacidad está configurada a través de la línea de comando al iniciar el software del _monerod_. Parámetros específicos de inicio se ilustran más abajo en el [Ejemplo #2](https://www.monerooutreach.org/stories/RPC-Pay.html#box2). De manera similar, hay comandos con el monedero CLI que lo habilitan para crear hashes y pagarle al monerod. Estos también están representados en el [Ejemplo #2](https://www.monerooutreach.org/stories/RPC-Pay.html#box2).
 
-With the current Monero release, there is a _monerod_ command (of the type that can be typed right into the _monerod_ console) related to RPC, rpc_payments. This command displays information on paying clients and their balances in the console window. Note that a node will forget about credit balances after they are untouched for six months.
+Con el lanzamiento actual de Monero, hay un comando en el _monerod_, rpc_payments, relacionado con el RPC (del mismo tipo de comandos que se pueden escribir directamente a la consola del _monerod_). Este comando muestra información de los clientes que pagan y sus balances en la ventana de la consola. Ten en cuenta que el nodo olvidará los balances de los créditos despues de no haber sido tocados por un periodo de seis meses.
+-------------
+RPC-Pay ofrece muchas ventajas. Apoya a la descentralización porque muchas personas querrán ejecutar un servidor que se pague a si mismo, versus un menor número de personas que querrán asumir el costo de ejecutar uno. RPC-Pay para el RPC también estimula a los usuarios de los monederos para que ejecuten su propio nodo para evitar pagar. Todo esto fortalece y descentraliza la red.
 
-RPC-Pay offers many advantages. It supports decentralization because many people will want to run a server that pays for itself, versus the smaller number willing to run one at cost. Payment for RPC also encourages some wallet users to run their own node to avoid having to pay. All this strengthens and decentralizes the network.
+RPC-Pay es privado. No se guardan grandes balances con créditos que requieran de compartir información para copias de seguridad y por temas de seguridad. Los hashes de minería usados para los pagos son anónimos, sin la posibilidad de rastrear su propiedad en la cadena de bloques. Para rastrear y guardar balances, una identificación de pagos (payment ID) puede ser añadida, pero con un procedimiento operativo basado en pequeños balances. Este payment ID puede ser cambiado con regularidad o definitivamente no ser usado.
```

Comment:
_No se guardan grandes balances de crédito que requieran de compartir información **para respaldos o por seguridad**_

_Una identificación de pagos (payment ID) puede ser usada para rastrear y guardar balances, pero con un procedimiento operativo basado en pequeños balances, esta identificación puede ser cambiada con regularidad o definitivamente no ser usada._

Reply:
Agree on this, I remember it took me a lot of time trying to resolve how it was going to be.